.github/workflows/sdk-client-ci.yml

```diff
@@ -13,8 +13,7 @@ jobs:
   build-and-test:
     strategy:
       matrix:
-        # TODO: Figure out mechanism to skip iOS framework on ubuntu and windows and re-add ubuntu and windows
-        os: [macos-latest]
+        os: macos-latest
```

Comment:
For reviewers: Not necessary since macos dotnet build can build the other platforms.